controllers/cloud.redhat.com/providers/kafka/strimzi.go

```diff
@@ -160,13 +150,10 @@ func (s *strimziProvider) configureKafkaCluster() error {
 
 	k.Spec = &strimzi.KafkaSpec{
 		Kafka: strimzi.KafkaSpecKafka{
-			Config:   &kConfig,
-			Version:  &version,
-			Replicas: replicas,
-			Resources: &strimzi.KafkaSpecKafkaResources{
-				Requests: &kRequests,
-				Limits:   &kLimits,
-			},
+			Config:    &kConfig,
+			Version:   &version,
+			Replicas:  replicas,
+			Resources: &s.Env.Spec.Providers.Kafka.Cluster.Resources,
```

Comment:
Changing this means we have no defaults set at all though right?

Reply:
Remember, the `Spec.Resources` was getting overwritten at a later step of the code (line 223 in the old code)

You're right, I think at the moment (and even before this PR) there was no concept of a default Resources value. So I'll add that.

Reply:
Added defaults, take a look